query-engine/core/src/schema_builder/input_types/input_fields.rs

```diff
@@ -272,3 +252,51 @@ where
     )
     .optional()
 }
+
+pub(crate) fn scalar_list_input_field_mapper<T>(
+    ctx: &mut BuilderContext,
+    model_name: String,
+    input_object_name: T,
+    f: ScalarFieldRef,
+    include_set_only: bool,
```

Comment:
If you want to distinguish between update and create operation sets, then I suggest naming it accordingly.
`is_create` or similar, for example.